app/modules/Documentation/services/Article.php

```diff
@@ -0,0 +1,234 @@
+<?php
+
+/**
+ * This file is part of Vegas package
+ *
+ * @author Tomasz Borodziuk <tomasz.borodziuk@amsterdam-standard.nl>
+ * @copyright Amsterdam Standard Sp. Z o.o.
+ * @homenavigation https://bitbucket.org/amsdard/vegas-phalcon
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Documentation\Services;
+
+use Documentation\Models\Article as ArticleModel;
+use Phalcon\DI\InjectionAwareInterface;
+use Vegas\DI\InjectionAwareTrait;
+
+class Article implements InjectionAwareInterface
+{
+    use InjectionAwareTrait;
+    
+    private $serviceVersion = null;
+    private $serviceCategory = null;
+
+    private function getServiceVersion()
+    {
+        if($this->serviceVersion == null) {
```

Comment:
When serviceVersion = 0, this will give true. Please use one of following ways of testing:
- isset()
- ===
- is_null()